components-registry-automation/src/test/kotlin/org/octopusden/octopus/components/registry/automation/ApplicationTest.kt

```
@@ -0,0 +1,222 @@
+        .start()
+        .waitFor()
+
+    companion object {
+
+        private const val CORRECT_EXIT_CODE = 0
+        private const val INCORRECT_COMMAND_EXIT_CODE = 1
+        private const val TEST_TARGET_PATH = "downloaded_copyrights/COPYRIGHT"
+        private const val HELP_OPTION = "-h"
+
+        private val componentsRegistryServiceHost = System.getProperty("test.components-registry-service-host")
+            ?: throw Exception("System property 'test.components-registry-service-host' must be defined")
```

detekt (1.23.8)

[warning] Exception is a too generic Exception. Prefer throwing specific exceptions that indicate a specific error case.

(detekt.exceptions.TooGenericExceptionThrown)